nix/sources.json

```diff
@@ -1,15 +1,15 @@
 {
     "chain-main": {
-        "branch": "master",
+        "branch": "jt/ibc-go-v10",
```

Comment:
Its already added in master?

Reply:
Yes, but the resource file requires a tarball to run tests, so we need to have a release build, which does not yet exist in the releases.